### ISSUE TITLE
Keras CNN Classifier

### ISSUE THREAD

**Issue by jdpigeon** [NeuroTechX/neurodoro#11]
Neural network approach for best performance and potential integration into app.

Inception and VGG16 both do a good job classifying eyes open vs eyes closed EEG spectrograms. In order to make this work for Neurodoro data we'll need to 

1. Tweak CNN input so that it takes numerical data rather than images

2. Figure out how to take a pretrained image classification CNN from classification to regression 
 - rewrite final few layers?
 - train on examples with difficulty/performance labels?

3. Train & tweak

**Comment by jdpigeon** [comment]
For datasets I would recommend [MonicaGRawEEG0.csv](https://github.com/NeuroTechX/neurodoro/blob/master/muse-data/MonicaGRawEEG0.csv). It's the longest session of Neurodoro data anyone has ever collected at one time :open_mouth:
